SnailTests/ClosureTests.swift

```diff
@@ -0,0 +1,21 @@
+//  Copyright © 2016 Compass. All rights reserved.
+
+import Foundation
+import XCTest
+@testable import Snail
+
+class ClosureTests: XCTestCase {
+    func testClosure() {
+        let expectation = self.expectation(description: "Closure Executed")
```

Comment:
Do you need `expectation` here? is anything async?

Reply:
I can check if closure is `not nil` if that works for you

Reply:
Updated to not use expectation